rules/aip0123/resource_reference_type_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0123
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestResourceReferenceType(t *testing.T) {
+	annotation := ` [(google.api.resource_reference) = {
+		type: "library.googleapis.com/Author"
+	}]`
+	for _, test := range []struct {
+		name       string
+		Type       string
+		Annotation string
+		problems   testutils.Problems
+	}{
+		{"Valid", "string", annotation, nil},
```

Comment:
Add a valid case for `repeated string` as well.

Reply:
Done.